custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -31,6 +31,7 @@ def validate_phone_number(string_value):
     if string_value is None or string_value.strip() in ['', '0']:
         return ''
     else:
+        assert int(string_value) > 0
         phone_number = str(int(string_value))
```

Comment:
Oh, that apparently would also suffice for the `> 0` check:
```python
In [1]: "-123".isdigit()
Out[1]: False
```
>  Return True if all characters in the string are digits and there is at least one character, False otherwise.